CHANGELOG.md

```diff
@@ -2,6 +2,114 @@ TODO: This document was manually maintained so might be incomplete. The
 automation effort is tracked in
 https://github.com/kubernetes/client-go/issues/234.
 
+# v6.0.0
+
+**Breaking Changes:**
+
+* Swagger 1.2 retriever was removed from the discovery client
+
+    * [https://github.com/kubernetes/kubernetes/pull/53441](https://github.com/kubernetes/kubernetes/pull/53441)
+
+* [k8s.io/apimachinery] `ObjectCopier` interface was removed (requires switch to new generators with DeepCopy methods)
+
+    * [https://github.com/kubernetes/kubernetes/pull/53525](https://github.com/kubernetes/kubernetes/pull/53525)
+
+* [k8s.io/api] Renamed `core/v1.ScaleIOVolumeSource` to `ScaleIOPersistentVolumeSource`
+
+    * [https://github.com/kubernetes/kubernetes/pull/54013](https://github.com/kubernetes/kubernetes/pull/54013)
+
+* [k8s.io/api] Renamed `core/v1.RBDVolumeSource` to `RBDPersistentVolumeSource`
+
+    * [https://github.com/kubernetes/kubernetes/pull/54302](https://github.com/kubernetes/kubernetes/pull/54302)
+
+* [k8s.io/api] Removed `core/v1.CreatedByAnnotation`
+
+    * [https://github.com/kubernetes/kubernetes/pull/54445](https://github.com/kubernetes/kubernetes/pull/54445)
+
+* [k8s.io/api] Renamed `core/v1.StorageMediumHugepages` to `StorageMediumHugePages`
+
+    * [https://github.com/kubernetes/kubernetes/pull/54748](https://github.com/kubernetes/kubernetes/pull/54748)
+
+* [k8s.io/api] Renamed `core/v1.StorageMediumHugepages` to `StorageMediumHugePages`
+
+    * [https://github.com/kubernetes/kubernetes/pull/54748](https://github.com/kubernetes/kubernetes/pull/54748)
+
+* Informers got a NewFilteredSharedInformerFactory to e.g. filter by namespace
+
+    * [https://github.com/kubernetes/kubernetes/pull/54660](https://github.com/kubernetes/kubernetes/pull/54660)
+
+* [k8s.io/api] The dynamic admission webhook is split into two kinds, mutating and validating.
```

Comment:
... For that the kinds have changed completely and old code must be ported to `admissionregistration.k8s.io/v1beta1` `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration`.

Reply:
Done.